floris/simulation/turbine.py

```diff
@@ -359,17 +561,22 @@ class Turbine(BaseClass):
             Defaults to 0.5.
     """
 
-    turbine_type: str
+    turbine_type: str = field()
     rotor_diameter: float = field()
-    hub_height: float
-    pP: float
-    pT: float
-    TSR: float
-    generator_efficiency: float
-    ref_density_cp_ct: float
+    hub_height: float = field()
+    pP: float = field()
+    pT: float = field()
+    TSR: float = field()
+    generator_efficiency: float = field()
+    ref_density_cp_ct: float = field()
+    ref_tilt_cp_ct: float = field()
     power_thrust_table: PowerThrustTable = field(converter=PowerThrustTable.from_dict)
-
-
+    # floating_tilt_table: NDArrayFloat = field(
```

Comment:
It looks like this is initialized below so this commented code is probably out of date